RELEASE-NOTES.rst

```diff
@@ -19,7 +19,7 @@ What's new
 Installation
 ------------
 
-   $ pip install invenio-db==1.0.0
+   $ pip install invenio-db==1.0.0a7
```

Comment:
I think it's nice to have the full version in case people use copy-and-paste for example. For us, `git grep 1.0.0a` is quick finding these anyway when times come to update them.  IIRC we used this for some of the past releases, perhaps we should harmonise habits.   (And indeed not updating changelog in the alpha stage...)